lib/oli_web/live/collaboration_live/collab_space_view.ex

```diff
@@ -120,53 +130,83 @@ defmodule OliWeb.CollaborationLive.CollabSpaceView do
 
   def render(assigns) do
     ~F"""
-      {#if show_collab_space?(@collab_space_config)}
-        {render_modal(assigns)}
-
-        <div class="card">
-          <div class="card-body d-flex align-items-center">
-            <h3 class="card-title">Collaborative Space</h3>
+    {render_modal(assigns)}
```

Comment:
Just to understand... If the collab space is disabled, will this be displayed anyway or that it will be displayed?  I think we should contemplate the possibility that the collab space is disabled, WDYT?

Reply:
We already discussed this online but it's going to be done [here](https://github.com/Simon-Initiative/oli-torus/pull/3287/files#diff-ee74bc1fa45f0e681c16090db0f9135eb587164e4eea0fc999c97ae88fa5fa01R43) now 😀